ansible_collections/arista/avd/examples/single-dc-l3ls/README.md

```diff
@@ -0,0 +1,831 @@
+2. Defines the scope of where these endpoints are connected to the fabric, in this case the two groups `DC1_L3_LEAVES` and `DC1_L2_LEAVES`.
+
+## Defining device types
+
+Since we're building a L3LS network, we must tell Ansible AVD which devices are spines, L3 leaves, L2 leaves etc. We have already grouped the devices in the inventory and so in this example this is done using the group variable files with the following names and content:
+
+`ansible-avd-examples/single-dc-l3ls/group_vars/DC1_SPINES.yml`
+
+```yml
+---
+type: spine
+```
+
+`ansible-avd-examples/single-dc-l3ls/group_vars/DC1_L3_LEAVES.yml`
+
+```yml
+---
+type: l3leaf
+```
+
+`ansible-avd-examples/single-dc-l3ls/group_vars/DC1_L2_LEAVES.yml`
+
+```yml
+---
+type: l2leaf
+```
+
+For example, this means that all switches that are children of the DC1_SPINES group defined in the inventory, will be of the type `spine`.
+
+## Setting fabric-wide configuration parameters
+
+In the `ansible-avd-examples/single-dc-l3ls/group_vars/FABRIC.yml` file, we define generic settings that apply to all children of the `FABRIC` group as defined in the inventory described earlier.
+
+The first section defines how the Ansible host connects to the devices:
+
+```yml
+ansible_connection: ansible.netcommon.httpapi
+ansible_network_os: arista.eos.eos
+ansible_user: ansible
+ansible_password: ansible
+ansible_become: true
+ansible_become_method: enable
+ansible_httpapi_use_ssl: true
+ansible_httpapi_validate_certs: false
+```
+
+In this case we're specifying that:
+
+- The Ansible host must use eAPI via HTTPS.
+- Network OS which in this case is Arista EOS.
+- The username/password combo.
+- How to escalate privileges to get write access.
+- We're using SSL.
+- We're not validating certs.
+
+In the next section we specify variables that generate configuration to be applied to all devices in the fabric:
+
+```yml
+fabric_name: FABRIC
+
+underlay_routing_protocol: EBGP
+overlay_routing_protocol: EBGP
+
+local_users:
+  ansible:
+    privilege: 15
+    role: network-admin
+    sha512_password: $6$7u4j1rkb3VELgcZE$EJt2Qff8kd/TapRoci0XaIZsL4tFzgq1YZBLD9c6f/knXzvcYY0NcMKndZeCv0T268knGKhOEwZAxqKjlMm920
+  admin:
+    privilege: 15
+    role: network-admin
+    no_password: true
+
+bgp_peer_groups:
+  evpn_overlay_peers:
+    password: Q4fqtbqcZ7oQuKfuWtNGRQ==
+  ipv4_underlay_peers:
+    password: 7x4B4rnJhZB438m9+BrBfQ==
+  mlag_ipv4_underlay_peer:
+    password: 4b21pAdCvWeAqpcKDFMdWw==
+
+p2p_uplinks_mtu: 1500
+```
+
+- The name of the fabric for internal Ansible AVD use. This name *must* match the name of an Ansible Group covering all network devices.
+- Local users/passwords and their privilege levels. In this case the "ansible" user is set with the password "ansible".
+- BGP peer groups and their passwords (all passwords are "arista").
+- Point-to-point interface MTU, in this case it's set to 1500 since the example is using vEOS, but when using hardware this should be set to 9214 instead.
```

Comment:
```suggestion
- Point-to-point interface MTU, in this case it is set to 1500 since the example is using vEOS, but when using hardware this should be set to 9214 instead.
```